### PR TITLE
refactor: move calendar button to the internal package

### PR DESCRIPTION
**Pull request by eduramme**

### CODE REVIEW

File: apps/namehashlabs.org/components/2 - molecules/contact-us-section.tsx [hunk 1]
```diff
@@ -28,7 +23,7 @@ export const ContactSection = () => {
           </p>
         </div>
         <div className="w-full md:w-auto">
-          <CalendarButton variant="secondary" className="!w-full">
```
Comment (lightwalker-eth) [inline apps/namehashlabs.org/components/2 - molecules/contact-us-section.tsx:31]: There are some screen widths where the layout here breaks. Perhaps it is related to "!w-full" being removed? I'm not familiar with what that does :)

CleanShot 2024-10-21 at 17 11 58

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@eduramme Hey overall looks good! Just 1 small change request :+1:

File: apps/namehashlabs.org/components/2 - molecules/contact-us-section.tsx (hunk 1 above)
Comment (eduramme) [reply apps/namehashlabs.org/components/2 - molecules/contact-us-section.tsx:31]: @lightwalker-eth done! :)

**Review by lightwalker-eth — APPROVED** [review]
Looks good 🙌

**Pull request merged**